### PR TITLE
[VIT-4446] Update docs with new clientfacingintervalsample fields

### PR DESCRIPTION
**Pull request by ItachiEU**
(and more I guess)

### CODE REVIEW

**Review by andersio — APPROVED** [review]

**Comment by ItachiEU** [comment]
rebase nightmare 😱

File: docs/api-reference/sleep/get-stream.mdx [hunk 1]
```diff
@@ -104,37 +104,47 @@ data = await client.Sleep.getStream(sleep_id=*)
     {
       "id": 0,
       "timestamp": "2022-08-04T12:42:45.722168+00:00",
+      "start": "2022-08-04T12:42:45.722168+00:00",
+      "end": "2022-08-04T12:43:15.722179+00:00",
       "value": 1,
-      "type": "vital_hypnogram",
-      "unit": "vital_hypnogram"
+      "type": "None",
```
Comment (andersio) [inline docs/api-reference/sleep/get-stream.mdx:110]: These should be JSON null 🤔

Comment (ItachiEU) [reply docs/api-reference/sleep/get-stream.mdx:110]: aaah, will fix

Comment (ItachiEU) [reply docs/api-reference/sleep/get-stream.mdx:110]: https://github.com/tryVital/docs/pull/139

**Pull request merged**